src/js/common/components/Modal.jsx

```diff
@@ -78,7 +78,7 @@ class Modal extends React.Component {
     }
 
     return (
-      <div className={modalCss} id={id} role="alertdialog" aria-labelledby={`${id}-title`}>
+      <div className={modalCss} id={id} role="alertdialog" aria-labelledby={id && title ? `${id}-title` : undefined}>
```

Comment:
Oh, so you're saying if React sees a div with `aria-labelledby=undefined` it will throw away the entire attribute? That seems more palatable.

I'm really curious to understand what kinds of things the 508 office does/doesn't catch, and similarly what our stricter tests can/can't test....it's all a bit mysterious since I haven't yet seen a comprehensive report from the 508 office about a particular product. Hopefully will soon.